user-operator/src/main/java/io/strimzi/operator/user/model/acl/SimpleAclRuleResource.java

```diff
@@ -10,11 +10,13 @@
 import io.strimzi.api.kafka.model.AclRuleResource;
 import io.strimzi.api.kafka.model.AclRuleTopicResource;
 
+import io.strimzi.api.kafka.model.AclRuleTransactionalIDResource;
```

Comment:
This import should be probably on line 12 together with other strimzi imports and the line 13 should be empty.